dspace-api/src/main/java/org/dspace/google/GoogleAsyncEventListener.java

```diff
@@ -171,9 +173,33 @@ private String getDocumentPath(HttpServletRequest request) {
         return documentPath;
     }
 
-    private boolean isNotBitstreamViewEvent(UsageEvent usageEvent) {
-        return usageEvent.getAction() != UsageEvent.Action.VIEW
-            || usageEvent.getObject().getType() != Constants.BITSTREAM;
+    /**
+     * Verifies if the usage event is a content bitstream view event, by checking if:<ul>
+     * <li>the usage event is a view event</li>
+     * <li>the object of the usage event is a bitstream</li>
+     * <li>the bitstream belongs to one of the configured bundles (fallback: ORIGINAL bundle)</li></ul>
+     */
+    private boolean isContentBitstream(UsageEvent usageEvent) {
+        // check if event is a VIEW event and object is a Bitstream
+        if (usageEvent.getAction() == UsageEvent.Action.VIEW
+            || usageEvent.getObject().getType() == Constants.BITSTREAM) {
```

Comment:
This logic looks wrong.  It's checking if it's a `VIEW` OR a Bitstream.  I think it should be using `&&`?